src/ScrumBoardItBundle/Resources/config/services.yml

```diff
@@ -4,9 +4,15 @@ services:
     jira_token_authenticator:
         class: ScrumBoardItBundle\Security\JiraAuthenticator
         arguments: ['@router','%jira%','@security.rememberme.response_listener']
```

Comment:
wouldn't it be nicer to have all the jira config gathered?
`jira:
  token_authenticator:
    class: ...
  api:
    class: ...`